migrations/2024-07-15-111327_change_primary_key_for_merchant_account/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+-- The below query will lock the merchant account table
+-- Running this query is not necessary on higher environments
+-- as the application will work fine without these queries being run
+-- This query is necessary for the application to not use id in update of merchant_account
+ALTER TABLE merchant_account DROP CONSTRAINT merchant_account_pkey;
```

Comment:
Mention a command which says this needs to ran after the deployments in environments